File: kods_C_C++/PYTHON/sentimental-mario-more/mario.py
import cs50
from cs50 import get_int


def main():

    height = get_height()

    for x in range(height):
        for spaces in range(height-1, x, -1):
            print(" ", end="")

        for hash_left in range(x+1):
            print("#", end="")

        # Middle space
        print("  ", end="")

        for hash_right in range(x+1):
            print("#", end="")

        print()


def get_height():
    while True:
        # height = get_int("Height: ")
        try:
            height = int(input("Height: "))
            if height > 0 and height < 9:
                break
        except ValueError:
            print("Not an int")

    return height


main()

File: kods_C_C++/ps1/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height;
    do
    {
        height = get_int("Height:  \n");
    } while (height < 1 || height > 8);



    for (int i = 0; i < height; i++)
    {
        for (int spaces = height - 1; spaces > i; spaces--)
        {
            printf(" ");
        }

        for (int hash_l = -1; hash_l < i; hash_l++)
        {
            printf("#");
        }

        // Printing gap:
        printf("  ");

        for (int hash_r = -1; hash_r < i; hash_r++)
        {
            printf("#");
        }
        printf("\n");
    }
}
